src/main.js

```diff
@@ -1300,14 +1300,8 @@ function appendArtworkCardsFromShardsAndItems(shards, items, proxy, type, direct
                         title: entry.title,
                         author: entry.author,
                         thumbnail: thumbnail,
-                        desktop_id: proxy.desktopId,
-                        bus_name: proxy.busName,
-                        knowledge_search_object_path: proxy.knowledgeSearchObjectPath,
-                        knowledge_app_id: proxy.knowledgeAppId,
-                        uri: entry.ekn_id,
```

Comment:
Hmm, this is a good point (the perils of me copy-pasting code...). That being said, I wonder if artwork cards should "go" somewhere when clicked, to the WikiArt app, perhaps? In that case, `DiscoveryFeedKnowledgeArtworkCardStore` should be a subclass of `DiscoveryFeedKnowledgeAppCardStore` ?